commons/src/main/java/gov/cms/qpp/conversion/DocumentationReference.java

```diff
@@ -12,10 +12,10 @@ public enum DocumentationReference {
 	MEASURE_REFERENCE(36),
 	CEHRT(22);
 
-	private static final String BASE_PATH = "https://ecqi.healthit.gov/sites/default/files/2024-CMS-QRDA-III-EC-IG-v1.1-508.pdf#page=";
-	public static final String PERFORMANCE_START_DATE = "01/01/2024";
-	public static final String PERFORMANCE_END_DATE = "12/31/2024";
-	public static final String ROSTER_UPDATE_DATE = "December 13, 2024";
+	private static final String BASE_PATH = "https://ecqi.healthit.gov/sites/default/files/2025-CMS-QRDA-III-EC-IG-v1.1.pdf#page=";
+	public static final String PERFORMANCE_START_DATE = "01/01/2025";
+	public static final String PERFORMANCE_END_DATE = "12/31/2025";
+	public static final String ROSTER_UPDATE_DATE = "December 13, 2025";
```

Comment:
Just a note.  Roster date changes every year.  You need to check with Steven or Matt Svenby to find this year roster update by date.  In 2023, it was Dec 1.  Not a breaking change.

Reply:
will ask and update this once we get